src/tracing/errors.ts

```diff
@@ -93,6 +93,11 @@ export const filterKnownErrors: Required<ClientOptions>['beforeSend'] = (event:
     if (error.message.match(/'unsafe-eval'.*content security policy/i)) {
       return null
     }
+
+    // WebAssembly compication requires 'unsafe-eval' in CSP, meaning these errors can be filtered out.
```

Comment:
This comment is unclear. What about:

```suggestion
    // WebAssembly compilation fails because we do not allow 'unsafe-eval' in our CSP.
    // Any thrown errors are due to 3P extensions/applications, so we do not need to handle them.
```